youtube_transcript_api/_api.py

```diff
@@ -99,47 +99,39 @@ def get_transcript(cls, video_id, languages=None, proxies=None):
 
 class _TranscriptFetcher():
     WATCH_URL = 'https://www.youtube.com/watch?v={video_id}'
-    API_BASE_URL = 'https://www.youtube.com/api/{api_url}'
+    API_BASE_URL = 'https://www.youtube.com/api/'
     LANGUAGE_REGEX = re.compile(r'(&lang=.*&)|(&lang=.*)')
     TIMEDTEXT_STRING = 'timedtext?v='
 
     def __init__(self, video_id, languages, proxies):
         self.video_id = video_id
         self.languages = languages
         self.proxies = proxies
+        self.matched_splits = []
 
     def fetch(self):
```

Comment:
General question: This way of filtering does also ignore whether a transcript was automatically generated or not, right? 🤔

Reply:
I'm not quite sure what you mean by this, but it should return the non-asr transcripts before asr ones.

Reply:
That is exactly what I mean, but it was already answered by your reply to the other comment! 👍